editor/src/components/documentation/release-notes.tsx

```diff
@@ -3,33 +3,50 @@ import { jsx } from '@emotion/react'
 import styled from '@emotion/styled'
 
 import * as React from 'react'
-import { colorTheme } from '../../uuiui'
+import { colorTheme, SimpleFlexColumn, SimpleFlexRow, UtopiaTheme } from '../../uuiui'
+import { FileTabs } from '../filebrowser/file-tabs'
 import { H1, H2, PrettyKeys, EM, CalloutPrimary, A } from './documentation-components'
 
 import { GettingStarted } from './getting-started'
 
 export function ReleaseNotesContent() {
   return (
-    <div
-      css={{
-        label: 'ReleaseNotesContainer',
-        backgroundColor: colorTheme.emphasizedBackground.value,
-        fontSize: '16px',
-        lineHeight: '26px',
-        paddingBottom: 18,
-        width: '100%',
-        cursor: 'text',
-        userSelect: 'text',
-        WebkitUserSelect: 'text',
-        overflow: 'scroll',
-        whiteSpace: 'pre-wrap',
-        '&  *': {
+    <SimpleFlexColumn>
+      <SimpleFlexRow
```

Comment:
this SimpleFlexRow with the same style object is used in two places – could you make it part of FileTabs to avoid copy-pasted stlye code?

Reply:
fixed